logger/logger.go

```diff
@@ -113,6 +115,21 @@ func SetAutoFormat() {
 	}
 }
 
+// SetAutoJSONFormat uses the environment variable `LOG_COLOR` and terminal detection to select console or JSON output format
+func SetAutoJSONFormat() {
```

Comment:
Is this function actually called from anywhere?

Reply:
Not yet, it's meant to replace SetJSONFormat so we can still see colored logs during development